frontend/index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Skin Crawler</title>
+	<link rel="stylesheet" type="text/css" href="style.css">
+	<link rel="preconnect" href="https://fonts.gstatic.com">
+	<link href="https://fonts.googleapis.com/css2?family=Roboto+Mono&display=swap" rel="stylesheet">
+</head>
+<body>
+	<div class="header">
+		<a href="../index.html" class="header__title">Skin Crawler</a>
+		<div class="header__rec"></div>
+		<div class="header__ellipse"></div>
+	</div>
+  <section class="container" id="inf-img">
+    <script>
+        function addImage() {
+            var data = JSON.parse('{{ data | tojson | safe}}');
```

Comment:
We aren't using Jinja here. I suppose data can be passed in the function as JSON, so this could be written like.

```suggestion
        function addImage(data) {
```